### ISSUE TITLE
Install errors out

### ISSUE THREAD

**Issue by durd** [Pack3tL0ss/ConsolePi#191]
Installing using the automated way errors out after selecting extras (I selected all extras if that makes a difference).

```
cat: /tmp/ansible_ver: No such file or directory
cat: /tmp/ansible_ver: No such file or directory
/etc/ConsolePi/installer/utilities.sh: line 207: hash: pipx: not found
Error: Unsupported flag passed to process_cmds --apt-install
```

As a workaround I installed pipx, manually via apt and ran the extra commands from that if-sequence below line 207.
There were some issues installing ansible and the ansible collections, but re-running the install worked.

**Comment by Pack3tL0ss** [comment]
@durd Thank you I thought I tested all that with a recent push, but I'll take another look.

**Comment by durd** [comment]
I'm sorry for the rushed issue, I should add that I'm running Bookworm on the RPi4. And the pipx issue seemed to be just a dash too many on the `--apt-install`.
I didn't have time to look into the Ansible collection thing, of course I didn't think about saving the messages from that :/

**Comment by Pack3tL0ss** [comment]
No worries, what you provided was enough to give me an idea of the issue, and it's exactly as you mentioned.  On the `bookworm` branch where I'm nearly done re-writing all the network automations to work with NetworkManager, I updated it so `-apt-install` or `--apt-install` would both work (double `--` seems more intuitive to me now for multi-letter flags).  Then I updated the `dev` branch to update the way ansible was installed, and merged it.  I did test it, but must have been on the wrong branch when I did.

Fix will be merged shortly.
